src/extensionFunctions.ts

```diff
@@ -182,6 +182,7 @@ export async function startTerminal(...rest: any[]): Promise<void>{ //name:strin
 export async function showAndSendText(terminal: vscode.Terminal, text:string): Promise<void> {
 	if (terminal) {
 		terminal.show();
+		utils.delay(100);
```

Comment:
I do not understand why there is still a delay intgroduced in the production code.

Reply:
I will try without it.